### ISSUE TITLE
bug: incorrect type for status field on get return

### ISSUE THREAD

**Issue by TillaTheHun0** [hyper63/hyper-adapter-queue#11]
the `Queue` port defines the argument and return type of `get` as these `zod` objects:

Arguments:
```
const QueueGetInput = z.object({
  name: z.string(),
  status: z.enum(["READY", "ERROR"]),
});
```

Return:
```
const JobsResponse = z.object({
  ok: z.boolean(),
  jobs: z.array(z.object({}).passthrough()).optional(),
  status: z.number().optional(),
});

```

get:
```
  get: z.function()
    .args(QueueGetInput)
    .returns(z.promise(JobsResponse)),
```

This adapter's `get` impl is:

```
  async function get({ name, status }) {
    return await db.find({ type: JOB, queue: name, status })
      .then((jobs) => ({ ok: T, jobs, status }));
  }
```

Notice that `status` arg is copied to the returned object. This causes a `zod` validation error like:
```
{
  "issues": [
    {
      "code": "invalid_return_type",
      "returnTypeError": {
        "issues": [
          {
            "code": "invalid_type",
            "expected": "number",
            "received": "string",
            "path": [
              "status"
            ],
            "message": "Expected number, received string"
          }
        ],
        "name": "ZodError"
      },
      "path": [
        "get"
      ],
      "message": "Invalid function return type"
    }
  ],
  "name": "ZodError"
}
```

Because `zod` expects a number, not a string. Either the `Queue` port is incorrect, or this adapter needs to return a number in the `status` field on the `get` adapter method.

**Comment by TillaTheHun0** [comment]
Per conversation with @twilson63, `status` in the response is meant to be a more general purpose status code, not a job status. So the adapter shouldn't be returning a status, unless a code is needed, usually to indicate an error.

**Comment by TillaTheHun0** [comment]
Closed by #12
